run.py

```diff
@@ -0,0 +1,174 @@
+import math
+import os
+import re
+import subprocess
+import random
+import shutil
+from loguru import logger
+from collections import defaultdict
+from distutils.dir_util import copy_tree
+from multiprocessing.dummy import Pool as ThreadPool, current_process
+bot = ["basic_bot", "T", "python"]
+
+opponents = [
+    ["loser_bot", "T", "python"],
+]
+
+maps_path = "D:/Games/StarCraft II/Maps"
```

Comment:
@ratosh I'll merge this file and then rename it to "multithread-run-example.py".
This way it will be clear it's an example file.

Then someone can submit a PR to refine it to work with all environments/maps/etc, if they want.